server/message.go

```diff
@@ -0,0 +1,57 @@
+package server
```

Comment:
If we split the reusable server code and the executable implementation into separate packages, I would include this module in the "reusable" package.